lib/services.js

```diff
@@ -476,6 +476,7 @@ export default {
             params.headers['content-type'];
         multipartObjectMD.expires =
             params.headers.expires;
+        multipartObjectMD.storageClass = params.storageClass;
```

Comment:
it is not the same as 'x-amz-storage-class' field ? it should be better to get the 'x-amz-storage-class' already store in metadata and if needed, map it into 'storageClass' field when we get metadata, in that way, any older values store will be used :)